tests/test_callprofiler.py

```diff
@@ -380,4 +380,47 @@ def foo():
         modmesh.call_profiler.reset()
         foo()
 
+    def test_status(self):
+        modmesh.wrapper_profiler_status.disable()
+        self.assertFalse(modmesh.wrapper_profiler_status.enabled)
+
+        modmesh.call_profiler.reset()
+        buf = modmesh.ConcreteBuffer(10)
+        result = modmesh.call_profiler.result()
+        self.assertEqual({}, result)
```

Comment:
Since TimeRegistry is return a list for result, CallProfiler return a dict for result. So the result should expected as a empty dict.
```diff
- self.assertEqual([], modmesh.time_registry.names)
+ result = modmesh.call_profiler.result()
+ self.assertEqual({}, result)
```